### PR TITLE
Update get-access-to-and-back-up-a-former-user-s-data.md

### PR DESCRIPTION
**Pull request by msbemba**
LIne 180- Added note to mention that only12 months of data is available on a new profile.
LIne 213- Added a note to mention that the steps remain the same for existing users as well.

### CODE REVIEW

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/1465  and #https://github.com/MicrosoftDocs/microsoft-365-docs/issues/1498

**Comment by mypil** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: microsoft-365/admin/add-users/get-access-to-and-back-up-a-former-user-s-data.md [hunk 1]
```diff
@@ -176,6 +176,10 @@ Outlook begins the export immediately unless a new Outlook Data File (.pst) is c
     
 See how to [Export or backup email, contacts, and calendar to an Outlook .pst file](https://support.office.com/article/14252b52-3075-4e9b-be4e-ff9ef1068f91.aspx) in Outlook 2010. 
   
+  
+  > [!NOTE]
+  > By default, 12 months of your email is available offline. See how to [Increase the data available offline](Https://docs.microsoft.com/en-us/outlook/troubleshoot/mailboxes/only-subset-items-synchronized)
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/add-users/get-access-to-and-back-up-a-former-user-s-data.md]:
```suggestion
  > By default, your email is available offline for a period of 12 months. If required, see how to [increase the data available offline](Https://docs.microsoft.com/outlook/troubleshoot/mailboxes/only-subset-items-synchronized).
```

File: microsoft-365/admin/add-users/get-access-to-and-back-up-a-former-user-s-data.md [hunk 2]
```diff
@@ -205,6 +209,9 @@ To give access to the email messages, calendar, tasks, and contacts of the forme
 8. Set the options for importing items. The default settings usually don't need to be changed.
     
 9. Select **Finish**.
+
+> [!NOTE]
+> The steps remain the same for accessing existing user Onedrive and email data.
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/add-users/get-access-to-and-back-up-a-former-user-s-data.md]:
```suggestion
> The steps remain the same for accessing an existing user's OneDrive and email data.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by mypil** [comment]
@msbemba - please merge the commit on the copy/edit review done by JohanFreelancer9 or make the necessary changes on the PR following the suggestions made. Thank you.

**Review by JohanFreelancer9 — APPROVED** [review]

**Pull request merged**